lib/bolt/target.rb

```diff
@@ -291,10 +291,11 @@ def detail
           'transport' => transport,
           transport => options
         },
-        'vars' => vars,
-        'facts' => facts,
+        # Dup frozen hashes to deep_cline in outputter
+        'vars' => vars.dup,
```

Comment:
I'm not sure I follow why `dup` is necessary here...

Reply:
deep_merge was throwing "can not modify frozen hash". We are just going with JSON.pretty_generate here anyway, so this will be removed.

Reply:
That's... puzzling. `deep_merge` shouldn't need to modify anything...

Reply:
Oh wait it's `deep_clone`. That makes more sense.